cadence/nft-lock-box/transactions/claim_nft.template.cdc

```diff
@@ -0,0 +1,36 @@
+import NonFungibleToken from {{{ imports.NonFungibleToken }}}
+import NFTLockBox from {{{ imports.NFTLockBox }}}
+import {{ contractName }} from {{{ contractAddress }}}
+
```

Comment:
It would be good to add a comment at the top of each transaction and script to describe what each is doing